Library/ViewModels/AddNewCardViewModelTests.swift

```diff
@@ -340,41 +335,43 @@ internal final class AddNewCardViewModelTests: TestCase {
 
   func testUnsupportedCardMessage_showsWithInvalidCardBrand_AndExistingCardNumber() {
     self.vm.inputs.viewDidLoad()
+    self.vm.inputs.configure(with: .pledge, project: Project.template)
```

Comment:
Can you clarify what `.intent` is being tested here? It seems like this should be configured with `project: nil`. We should also update the test name to describe what intent is being tested.

Reply:
Not sure what your feedback is here. .pledge intent is being tested here and that needs to be configured with a `project`